magento/ece-tools
# MAGECLOUD-1092: Move out ECE-Tools source code from src subdirectory

https://magento2.atlassian.net/browse/MAGECLOUD-1092

Depends on: https://github.com/magento/ece-tools/pull/44

### Description
This PR fixed unnecessary deep directory structure `src/Magento/MagentoCloud` by putting all code into `src` directory.

### Fixed Issues (if relevant)

1. Eliminated superfluous directory structure

### Manual testing scenarios

1. Deploy Magento
2. Verify if it works

### Contribution checklist
 - [ ] Pull request was approved by architect
